### PR TITLE
tests

### PR DESCRIPTION
**Pull request by katzio**

### CODE REVIEW

File: appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt [hunk 1]
```diff
@@ -84,12 +84,14 @@ class AppInfoPresenter(private val appName: String?, private val repository: Dis
                 view?.updateTransferStatus(TransferBarView.TransferStatus.FailedConnectionError)
             }
         }
+        // TODO: @Shaybaz What if not RESULT_OK?
```
Comment (katzio) [inline appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt]: @shayb

File: appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt [hunk 2]
```diff
@@ -157,12 +159,11 @@ class AppInfoPresenter(private val appName: String?, private val repository: Dis
                 if (!started) {
                     view?.updateTransferStatus(TransferBarView.TransferStatus.FailedReceiverError)
                 }
-            } ?: kotlin.run {
+            } ?: kotlin.run { //TODO: Remove, redounded, see L:40
```
Comment (katzio) [inline appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt]: @shayb

File: appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt [hunk 3]
```diff
@@ -18,9 +18,9 @@ import org.kinecosystem.common.utils.isAppInstalled
 class AppInfoPresenter(private val appName: String?, private val repository: DiscoveryAppsRepository, private val transferManager: TransferManager) : BasePresenter<IAppInfoView>(), IAppInfoPresenter {
 
 
-    private val AMOUNT_CHOOSER_REQUEST_CODE = 100
```
Comment (katzio) [inline appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt:21]: Private variables are used in a public method, untestable

File: appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt [hunk 4]
```diff
@@ -84,12 +84,14 @@ class AppInfoPresenter(private val appName: String?, private val repository: Dis
...
             }
         }
+        // TODO: @Shaybaz What if not RESULT_OK?
     }
 
     private fun sendKin(amountToSend: Int) {
         app?.let {
             it.identifier?.let { pkg ->
-                view?.initTransfersInfo(TransferInfo(repository.getStoredAppIcon(), it.iconUrl, it.name, pkg, amountToSend))
+                val info = TransferInfo(repository.getStoredAppIcon(), it.iconUrl, it.name, pkg, amountToSend)
```
Comment (shayb) [inline appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppInfoPresenter.kt]: change to transferInfo

**Pull request closed without merging**